packages/web-components/src/dialog-body/dialog-body.template.ts

```diff
@@ -25,7 +25,7 @@ export const template: ElementViewTemplate = html`
     <slot name="title"></slot>
     <slot name="title-action">
       <fluent-button
-        ?hidden=${x => x.noTitleAction || x.parentNode?.type !== DialogType.nonModal}
+        ?hidden=${x => x.noTitleAction || x.parentNode?.type !== DialogType.alert}
```

Comment:
I think this should be `===` - hide if noTitleAction is true OR parentNode is alert

```suggestion
        ?hidden=${x => x.noTitleAction || x.parentNode?.type === DialogType.alert}
```

Reply:
That it should indeed!